docs/2.guide/2.directory-structure/2.env.md

```diff
@@ -55,6 +55,10 @@ Since `.env` files are not used in production, you must explicitly set environme
 
 * Many cloud service providers, such as Vercel, Netlify, and AWS, provide interfaces for setting environment variables via their dashboards, CLI tools or configuration files.
 
+::warning
+If you are looking to update your environment variables in production, refer to [`runtimeConfig` docs](https://nuxt.com/docs/guide/going-further/runtime-config#environment-variabes) as using `process.env` in an attempt to read environment variables at runtime will not work.
+::
+
```

Comment:
I'm not sure if this might be misleading.

Some thoughts:
- Runtime config cannot be updated in production when using generate/static builds.
- There are some places where you can't use runtime config (outside of Nuxt context), where you have to use process.env.
- I think you should always use the runtimeConfig inside the Nuxt context.

Maybe we can simplify this to:
- Always use the runtimeConfig.
- If you are not in a Nuxt context, you can only use process.env.

Reply:
In confirmation to @huang-julien yes, `process.env` is guaranteed by nitro to be accessible at least in all server contexts we can also sync same env with context of nuxt config and modules.

Static build is exception.